index.js

```diff
@@ -0,0 +1,50 @@
+var playItem = require('./src/playitem.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var playlist = function playlist(list) {
+    this.playlist = playlistMaker(this, list);
+  };
+
+  playlist.list = function list() {
+    return plist.slice();
+  };
+
+  playlist.item = function item(index) {
```

Comment:
@heff I'm not adding method to the player itself. Modifying objects you don't own is a bad.